pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/TenantsBase.java

```diff
@@ -282,4 +284,17 @@ private CompletableFuture<Void> validateClustersAsync(TenantInfo info) {
             }
         });
     }
+
+    private CompletableFuture<Void> validateAdminRoleAsync(TenantInfoImpl info) {
+        // empty adminRole shouldn't be allowed
```

Comment:
Your implementation does not match this comment. If you mean `adminRoles`, in your implementation, it returns a normal completed future if it's empty. If you mean a single role from the `adminRoles` set, the comment should not be here and the code is clear enough.

So I suggest removing this comment.